### DISCUSSION TITLE
@parcel/transformer-js: Expression expected

### DISCUSSION THREAD

**Discussion by ys1113457623** [parcel-bundler/parcel#9180] [Q&A]
<img width="508" alt="Screenshot 2023-08-05 at 4 55 18 PM" src="https://github.com/parcel-bundler/parcel/assets/74351253/571c4715-6aff-48f2-86aa-46e3985d203d">
I am getting this error while running yarn parcel frontend/src/index.html --https

**BUT SYNTAX SEEMS TO BE CORRECT**

**Comment by mischnic** [comment]
For JSX to be correctly handled you need to
- use it in a `.jsx` file, or
- have `preact`/`react`/... as a dependency of your project, or
- (as a last resort: enable it explicitly in a tsconfig/jsconfig file)

See the docs as well: https://parceljs.org/languages/javascript/#jsx
